youtube.html

```diff
@@ -0,0 +1,256 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Youtube Clone</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="styles/general.css">
+    <link rel="stylesheet" href="styles/video.css">
+    <link rel="stylesheet" href="styles/header.css">
+</head>
+<body>
+    <nav class="header">
+        <div id="left-session">
+            <img id="menu" src="icons/menu.png" alt="">
+            <img id="youtube-icon" src="icons/youtube-icon.png" alt="">
+        </div>
+        <div id="side-bar">
+            <div id="side-bar-divs">
+                <img id="side-bar-home" src="icons/homepage.png" alt="">
+                <p id="side-bar-names">Home</p></div>
+            <div id="side-bar-divs">
+                <img id="side-bar-shorts" src="icons/youtube-shorts-icon.png" alt="">
+                <p id="side-bar-names">Shorts</p>
+            </div>
+            <div id="side-bar-divs">
+                <img id="side-bar-subscription" src="icons/subscribe.png" alt="">
+                <p id="side-bar-names">Subscriptions</p>
+            </div>
+            <div id="side-bar-divs">
+                <img id="side-bar-library" src="icons/video-library.png" alt="">
+                <p id="side-bar-names">Library</p>
+            </div>
+            
+            
+            
+            
+        </div>
+        <div id="middle-session">
+            <input type="text" name="" id="search-bar" placeholder="Search">
+            <button id="search-button">
+                <img id="search-icon" src="icons/search.png" alt="">
+            </button >
+            <button id="microphone">
+                <img id="voice-icon" src="icons/microphone.png" alt="">
+            </button>
+        </div>
+        <div id="right-session">
+            <img id="upload-icon" src="icons/upload.png" alt="">
+            <img id="video-icon" src="icons/video-player.png" alt="">
+            <img id="bell-notification" src="icons/bell.png" alt="">
+            <img id="profile-pic" src="profiles/channel-pic.jpg" alt="">
+        </div>
```

Comment:
Please consider using `<ul>` and `<li>` tags to display the items. Using `<ul>` and `<li>` tags gives meaning to the html nodes, which is very useful to screen readers and web crawlers and also helps increase the SEO of your page.